src/pages/AppLogin.vue

```diff
@@ -74,22 +74,29 @@
 
 <script>
 import { onMounted } from "vue";
+import { useRouter } from "vue-router";
 
 import useArcanaWallet from "../use/arcanaWallet";
 import useToast from "../use/toast";
 
 export default {
   setup() {
-    const { toastError } = useToast();
-    const { requestSocialLogin } = useArcanaWallet();
+    const router = useRouter();
+    const { toastSuccess, toastError } = useToast();
+    const { requestSocialLogin, setHook } = useArcanaWallet();
 
     onMounted(async () => {
       document.title = "Login | Arcana Demo";
+      setHook("connect", () => {
+        router.push("/my-files");
+      });
```

Comment:
@karthik-durai Is this code tested. I tried it on demo.dev.arcana.network and the problem still persists. I also recall trying to check if the 'connect' event is throw and didn't see it work.